integration_tests/tests/integrity/integrity_employee_count.sql

```diff
@@ -0,0 +1,89 @@
+{{ config(
+    tags="fivetran_validations",
+    enabled=var('fivetran_validation_tests_enabled', false)
+) }}
+
+with worker_history as (
+
+    select *
+    from {{ ref('stg_workday__worker_history') }}
+),
+
+worker_position_history as (
+
+    select *
+    from {{ ref('stg_workday__worker_position_history') }}
+),
+
+employee_daily_history as (
+
+    select *
+    from {{ ref('workday__employee_daily_history') }}
+),
+
+
+--to segment out workers who did not have a start date prior to the spine cutoff
+worker_last_start_date as (
+
+    select worker_id,  
+        source_relation,
+        cast(max(_fivetran_start) as date) as max_start_date
+    from worker_history
+    group by 1, 2
+),
+
+employee_source as (
+
+    select
+        worker_history.worker_id,
+        worker_history.source_relation,
+        worker_position_history.position_id,
+        worker_position_history.position_start_date
+    from worker_history 
+    join worker_last_start_date 
```

Comment:
Does this need to be a `left join` instead of a `join`? When I inspect these results I see that the test is passing, but not actually. 

When I run this in the database but comment out the final `where total_employee_source != total_employee_end` statement I get null records for the `total_employee_source` record.

Reply:
Thank you for catching. This integrity test took a lot of time to figure out the proper logic, with the surrogate key not making it easy to compare the source data with the end model.

I have reworked the logic and the test should be passing now.